conf/defaults.ini

```diff
@@ -1434,6 +1434,11 @@ enable_alpha = false
 app_tls_skip_verify_insecure = false
 # Enter a comma-separated list of plugin identifiers to identify plugins to load even if they are unsigned. Plugins with modified signatures are never loaded.
 allow_loading_unsigned_plugins =
+# Enter a comma-separated list of plugin identifiers to identify core data source plugins to be enabled explicitly. All are enabled if not specified or empty.
+# If not empty, those that are not given will be never loaded and not shown in plugin catalog. 
+core_datasource_plugins_enabled =
+# Hide Grafana Cloud phantom plugins including enterprise ones in data source adding page if true. By default, it is false.
+plugin_grafana_cloud_hidden =
```

Comment:
I think this should be an Grafana  enterprise setting, as it kind of relates to white labeling / oem